classification.yaml

```diff
@@ -0,0 +1,791 @@
+DMS-MWST-REQ-0030:
+  status: unclear
+  notes:
+    This is a pretty big feature request in terms of breaking assumptions we make in how PipelineTasks are ordered and related, and I'm not sure it's the right way to approach the problem of feeding DIAObjects to Alert Generation.
+    But we clearly need to think about how to solve that problem one way or another.
+DMS-MWST-REQ-0031:
+  status: unclear
+  notes:
+    I _think_ we might satisfy this via skipping quanta whose datasets exist in the current output RUN collection (using the same RUN collection is what guarantees config/software/input consistency).
+    But that's a very different description of the functionality from what's in the requirement.
+    Might want to check with GPDF to see if he buys this interpretation.
+    We might also be able to have more sophisticated skip-existing logic when provenance stuff has landed, which would more clearly satisfy the use case of the requirement even if it's not really more similar in terms of description.
+DMS-MWST-REQ-0004:
+  status: feature-complete
+  notes:
+    Pipeline YAML files satisfy this requirement (they don't even need to be ordered; the system can order them itself).
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0005:
+  status: feature-complete
+  notes:
+    Pipeline YAML files satisfy this requirement.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0006:
+  status: feature-complete
+  notes:
+    Grouping information is actually embedded in the PipelineTask definition, not directly the Pipeline, but since a Pipeline include PipelineTasks, this is still satisfied.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0007:
+  status: feature-complete
+  notes:
+    See notes for DMS-MWST-REQ-0006.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0008:
+  status: feature-complete
+  notes:
+    Satisfied by PipelineTask being a subclass of Task.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0003:
+  status: feature-complete
+  notes:
+    The Pipeline Python API is not heavily utilized, but it certainly does exist and was conceived of as a public interface.
+DMS-MWST-REQ-0010:
+  status: feature-complete
+  notes:
+    This is just how pipetask and BPS work.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0011:
+  status: feature-complete
+  notes:
+    This is just how pipetask and BPS work.
+  jira:
+  - LVV-T1982
+  - LVV-T1983f
+DMS-MWST-REQ-0012:
+  status: feature-complete
+  notes:
+    QuantumGraph generation guarantees this.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0013:
+  status: feature-complete
+  notes:
+    This is how pipetask and BPS work.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0009:
+  status: unclear
+  notes:
+    It looks like someone wants to delete this as redundant with -0013.  But we can do it all.
```

Comment:
Seems like this was already slated for deletion.

Reply:
Agree - remove